book/getting-started/setup.rst

```diff
@@ -95,14 +90,15 @@ in squared brackets in the following example:
                     </environment>
                     <environment type="dev">
                         <urls>
-                            <url>[url]</url>
-                            <url language="en" country="us">localhost</url>
+                            <url>[url]/{localization}</url>
```

Comment:
Actually not... These are the placeholders that have to be replaced by the user. We should actually think about updating this guide to use sulu-minimal, but that's a different story.